node/src/Transaction.ts

```diff
@@ -1968,6 +1969,22 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
             createGeoAdd(key, membersToGeospatialData, options),
         );
     }
+
+    /**
+     * Returns the positions (longitude, latitude) of all the specified `members` of the
+     * geospatial index represented by the sorted set at `key`.
+     *
+     * See https://valkey.io/commands/geopos for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param members - The members for which to get the positions.
+     *
+     * Command Response - A 2D `array` which represents positions (longitude and latitude) corresponding
+     *     to the given members. If a member does not exist, its position will be `null`.
```

Comment:
nit: looks like most of the docs don't indent the second line, not sure if this really matters though
```suggestion
     * to the given members. If a member does not exist, its position will be `null`.
```